cpp/ps0002/ps0002_sol.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+int main()
+{
+    short int people=100; // 0.5/0.75
+    short medecine=5; // 0.75/0.75
+    unsigned short number=50; // 0.75/0.75
+    float price=2.99; // 0.5/0.75
+    char first_character='n'; // 0.75/0.75
+    float energy_bill=55.5; // 0.75/0.75
+    unsigned int age_person=66666666; // 0.25/0.75
+    double number=7.546743; // 0.25/0.75
+    int china=1000000000; // 0.5/0.75
+    // total 5/6.75
+    return 0;
+}
+// good work 👍 6/10
```

Comment:
- [ ] program is in the correct path
- [ ] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code